deltachat-ffi/src/lib.rs

```diff
@@ -437,8 +438,8 @@ fn render_info(
     info: HashMap<&'static str, String>,
 ) -> std::result::Result<String, std::fmt::Error> {
     let mut res = String::new();
-    for (key, value) in &info {
-        writeln!(&mut res, "{}={}", key, value)?;
+    for key in info.keys().sorted() {
+        writeln!(&mut res, "{}={}", key, info.get(key).unwrap())?;
```

Comment:
Is `.get(key).unwrap()` the same as `[key]`? It may panic by the way.

Reply:
thought it could not panic as the key comes from the iterator the line above. but anyway, your BHashMap solution is much better!